qualang_tools/voltage_gates/README.md

```diff
@@ -106,20 +108,21 @@ In the example above:
 Add a compensation pulse of the specified duration whose amplitude is derived from the previous operations. This ensures that the average voltage seen by the gates remains zero over time.
```

Comment:
Now we don't specify the duration right?

Reply:
Yes good catch, I updated it!